### DISCUSSION TITLE
Retro Discussion

### DISCUSSION THREAD

**Discussion by decyjphr** [decyjphr-org/ghas-compliance#14] [Q&A]
There'll be 3 questions  
1. What went well
2. What could be better
3. What you are not sure about:

and you can answer each of them with mutliple answers which will be upvoted:

**Comment by decyjphr** [comment]
What went well

**Reply by decyjphr** [reply]
time tracking sucked

**Reply by decyjphr** [reply]
I was not well

**Comment by decyjphr** [comment]
What could be better?

**Comment by devasenab** [comment]
What I am not sure about
